ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml

```diff
@@ -25,7 +25,25 @@ resources:
       memory: 200Mi
 tests:
 - as: ci
-  commands: npm run build && npm run lint && npm test
+  commands: |
+    nodejs_version=v14.17.5
```

Comment:
can this be parametrized as a higher-level property outside the script?

Reply:
to do so, we could create a new make target that installs node, or else have that version in a file in the repo (I've seen this pattern used in other projects). any preference?
(we can do that in a future PR as anyway it requires also changes in the plugin repo)